modules/restore-replace-stopped-etcd-member.adoc

```diff
@@ -155,6 +155,7 @@ $ oc delete secret -n openshift-etcd etcd-serving-metrics-ip-10-0-131-183.ec2.in
 . Delete and recreate the control plane machine. After this machine is recreated, a new revision is forced and etcd scales up automatically.
 +
 If you are running installer-provisioned infrastructure, or you used the Machine API to create your machines, follow these steps. Otherwise, you must create the new master using the same method that was used to originally create it.
+//todo: assess for CPMS impact. 4.11: https://docs.openshift.com/container-platform/4.11/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-stopped-etcd-member_replacing-unhealthy-etcd-member
```

Comment:
Backup and restore > Control plane backup and restore > Replacing an unhealthy etcd member > [Replacing an unhealthy etcd member whose machine is not running or whose node is not ready](https://53397--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-stopped-etcd-member_replacing-unhealthy-etcd-member)

Reply:
Not sure what to do about this procedure. Is it still valid for a CMPS cluster that uses MHC?

Reply:
So this is a little tricky, what will happen today is the CPMS will react and create a replacement machine, but the user needs to manually remove the etcd machine deletion hook on the failed machine for the rollout to proceed, there is still a manual step here when the etcd member is not responding, which would be the case in these examples

Reply:
So, I _think_ that this is covered in the draft troubleshooting content (unreviewed so maybe wrong/incomplete, and unmerged but visible [here](https://53344--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-troubleshooting.html#cpmso-ts-mhc-etcd-degraded_cpmso-troubleshooting)). Would it make sense to add a note at the front of this procedure redirecting CPMS users to the CPMS docs?

Reply:
Yeah that could make sense, if you use CPMS, here's a better version

Reply:
ok cool, updated with a little note directing them to look for the easy way in the CPMS docs. Might be able to get a link in there if the TS PR merges first (or if I remember later). Als went and updated that TS topic to be less MHC specific, since I imagine that is merely one way in which etcd can become degraded.